### DISCUSSION TITLE
Visualizations

### DISCUSSION THREAD

**Discussion by trob** [trobfab/fabrik#29] [General]
I would like to have them back.
I have several Fabrik installations with vizs (fullcalendar, googlemaps) and would like to keep them (if we get Fabrik4 running ;) )

**Comment by skurvish** [comment]
I also need fullcalendar. Since I worked with Hugh originally on integrating this vis I will take the lead on getting it running. Once we get more of the front end working I should have time.

**Comment by skurvish** [comment]
fullcalendar is basically working. I think I would like to update to the latest version though.

**Comment by trob** [comment]
Why not, just try
